components/collective-page/sections/Contribute.js

```diff
@@ -239,7 +239,7 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const sortedTicketTiers = this.sortTicketTiers(this.filterTickets(tiers));
     const hideTicketsFromNonAdmins = (sortedTicketTiers.length === 0 || !collective.isActive) && !isAdmin;
-    const cannotOrderTickets = (!hasContribute && !isAdmin) || (!canOrderTicketsFromEvent(collective) && !isAdmin);
+    const cannotOrderTickets = (!hasContribute && !isAdmin) || isPastEvent(collective);
```

Comment:
Why are we not checking `canOrderTicketsFromEvent` anymore? What if there is no ticket?

Reply:
My understanding (and from testing this) is that the condition seems redundant, we already check for `sortedTicketTiers.length === 0` one line above and this is true when there's no tickets. So when there are no tickets the section is not shown anyways. Let me know if I am missing something. 🤔